lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -268,7 +266,8 @@ func (handler *ContainersHandlersImpl) GetContainerListHandler(params containers
 		info.ContainerID = &container.ExecConfig.ID
 		info.LayerID = &container.ExecConfig.LayerID
 		info.Created = &container.ExecConfig.Created
-		info.Status = &container.Status
+		s := container.State.String()
```

Comment:
Can you declare the string out of the loop and reuse the same variable instead of creating one for each vm?